front-end/src/Create.jsx

```diff
@@ -104,6 +104,8 @@ function Create() {
             if (stitchSongs.length != 0) {
                 const topSongImage = stitchSongs[0].album.images[0].url
                 updateImage(topSongImage)
+            } else {
+                updateImage('https://upload.wikimedia.org/wikipedia/commons/thumb/d/d1/Image_not_available.png/640px-Image_not_available.png')
```

Comment:
Probably better to extract this URL to a env var?

Reply:
I'm wary about the dependency we hold to this external resource. Can we have this as an asset? It can cease to exist at any point and the image would just fail to get retrieved.

Reply:
Got it, I'll download it in the asset folder and call it from there instead.